bip-emilengler-terminology-for-payment-recipient-identifiers.mediawiki

```diff
@@ -0,0 +1,49 @@
+<pre>
+  BIP: UNDEFINED
+  Title: Terminology for payment recipient identifiers
+  Author: Emil Engler <me@emilengler.com>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: CC0
+</pre>
+
+==Abstract==
+This BIP proposes a new term for 'address'
+
+==Specification==
+The new term is:
+''Bitcoin'' '''Invoice''' ''Address''
```

Comment:
Suggest specifying (vaguely) the intended sunset on the "address" suffix.

Reply:
Done